community_governance.adoc

```diff
@@ -381,77 +377,78 @@ If there are aspirational goals, those go in their own section under "Roadmap" o
-Are end-users members or just contributors?
-More than anything, defining Members means defining who it is that project leades need to listen to.
+Defining who Members are requires deciding who the project is actually serving, which is always a critical discussion to have.
+Are customers of the main sponsoring company automatically project Members?
+Can companies be Members, or only individuals?
+Are end-users Members or can they only be Contributors?
+More than anything, defining Members means defining who it is that project Leaders need to listen to.
 
-For almost all projects, you need to specify what rules members are subject to (usually a code of conduct and not much else) and what they can expect from leaders and contributors.
-It's particularly helpful to explain how members should participate in the project, such as "members file bugs against this repository, and use the 'new bug' template."
+For almost all projects, you need to specify what rules Members are subject to (usually a code of conduct and not much else) and what they can expect from Leaders and Contributors.
+It's particularly helpful to explain how Members should participate in the project, such as "Members file bugs against this repository, and use the 'new bug' template."
 Most people, given clear instructions, are happy to channel their participation into the routes you show them.
 
-In projects with democratically elected leadership, members can be a much more rigorously defined role, because being a member can come with voting rights.
-This requires you to more carefully qualify members to avoid vote-packing or simply derailing election procedures.
+In projects with democratically elected leadership, Members can be a much more rigorously defined role, because being a Member can come with voting rights.
+This requires you to more carefully qualify Members to avoid vote-packing or simply derailing election procedures.
 
 ==== Contributors
 
 Far more projects have a written definition of Contributors, but fewer than you'd think.
-It's often assumed, in the age of publicly hosted source code control, that you simply count up anyone who shows up in the Github or Gitlab statistics is therefore a Contributor.
+It's often assumed, in the age of publicly hosted source code control, that you count anyone in the GitHub or GitLab statistics as therefore a Contributor.
 But defining "who is a Contributor to this project" can be deceptively hard.
 
 Is it anyone who posted on a mailing list, or do you need 100 merged pull requests?
 Is it just code contributors, or contributors of any kind? What about folks who do events and advocacy?
-Are staff who work for a contributing company automatically considered contributors, or do they have to earn it individually?
+Are staff who work for a contributing company automatically consideredCcontributors, or do they have to earn it individually?
```

Comment:
Not sure what happened here?

Reply:
Inadvertent edit. Could be the drifting 'c' that my Bluetooth keyboard sometimes leaves.

I'm unsure what action to take here to fix this, or accept your requested changes. Time to experiment, starting with this comment to see what happens.

Reply:
fixed